config/param_types.go

```diff
@@ -396,3 +397,23 @@ func (p *OneofListParam) Parse(raw string) (result interface{}, err error) {
 	}
 	return
 }
+
+type CidrListParam struct {
+	Metadata
+}
+
+func (c *CidrListParam) Parse(raw string) (result interface{}, err error) {
+	value := strings.Split(raw, ",")
+	resultSlice := []string{}
+	for _, in := range value {
+		val := strings.Trim(in, " ")
+		if len(val) == 0 {
+			continue
+		}
+		_, _, e := cnet.ParseCIDROrIP(val)
+		if e != nil {
+			err = c.parseFailed(in, "invalid CIDR or IP")
+		}
```

Comment:
You don't append anything to "resultSlice";  suggest you take the output of ParseCIDROrIP and call String() on it so that you get a canonical CIDR.

Reply:
adding it to the `resultSlice` would be helpful... :-)
Good point about utilizing the `String()`.
let me formalize this PR today and have it ready for a complete review by tomorrow.